src/ai/backend/agent/docker/kernel.py

```diff
@@ -363,19 +363,13 @@ async def get_repl_out_addr(self) -> str:
         return f"tcp://{self.kernel_host}:{self.repl_out_port}"
 
 
-async def prepare_krunner_env_impl(distro: str) -> Tuple[str, Optional[str]]:
-    if distro.startswith("static-"):
-        distro_name = distro.replace("-", "_")  # pkg/mod name use underscores
-    else:
-        if (m := re.search(r"^([a-z]+)\d+\.\d+$", distro)) is None:
-            raise ValueError('Unrecognized "distro[version]" format string.')
-        distro_name = m.group(1)
+async def prepare_krunner_env_impl(distro: str, entrypoint_name: str) -> Tuple[str, Optional[str]]:
     docker = Docker()
     arch = get_arch_name()
     current_version = int(
         Path(
             pkg_resources.resource_filename(
-                f"ai.backend.krunner.{distro_name}", f"./krunner-version.{distro}.txt"
+                f"ai.backend.krunner.{entrypoint_name}", f"./krunner-version.{distro}.txt"
             )
         )
         .read_text()
```

Comment:
I found error log when Backend.AI agent is running.

```console
2023-02-13 16:41:19.400 DEBUG ai.backend.agent.docker.kernel [81314] loading krunner pkg: ai.backend.krunner.static_gnu.plugin
2023-02-13 16:41:19.402 ERROR aiotools.server [81314] Worker 0: Error during context manager initialization
  + Exception Group Traceback (most recent call last):
  |   File "/Users/dhsung/bai-dev/dist/export/python/virtualenvs/python-default/3.11.2/lib/python3.11/site-packages/aiotools/server.py", line 325, in _wrapped_worker
  |     async with ctx:
  |   File "/Users/dhsung/bai-dev/dist/export/python/virtualenvs/python-default/3.11.2/lib/python3.11/site-packages/aiotools/server.py", line 119, in __aenter__
  |     return (await self._agen.__anext__())
  |     ^^^^^^^^^^^^^^^^^
  |   File "/Users/dhsung/bai-dev/src/ai/backend/agent/server.py", line 680, in server_main_logwrapper
  |     async with server_main(loop, pidx, _args):
  |     ^^^^^^^^^^^^^^^^^
  |   File "/Users/dhsung/bai-dev/dist/export/python/virtualenvs/python-default/3.11.2/lib/python3.11/site-packages/aiotools/server.py", line 119, in __aenter__
  |     return (await self._agen.__anext__())
  |     ^^^^^^^^^^^^^^^^^
  |   File "/Users/dhsung/bai-dev/src/ai/backend/agent/server.py", line 714, in server_main
  |     krunner_volumes = await kernel_mod.prepare_krunner_env(local_config)  # type: ignore
  |       ^^^^^^^^^^^^^^^^^
  |   File "/Users/dhsung/bai-dev/src/ai/backend/agent/docker/kernel.py", line 483, in prepare_krunner_env
  |     async with TaskGroup() as tg:
  |       ^^^^^^^^^^^^^^^^^
  |   File "/Users/dhsung/bai-dev/dist/export/python/virtualenvs/python-default/3.11.2/lib/python3.11/site-packages/aiotools/taskgroup/base.py", line 40, in __aexit__
  |     raise TaskGroupError(eg.message, eg.exceptions) from None
  |     ^^^^^^^^^^^^^^^^^
  | aiotools.taskgroup.types.TaskGroupError: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/Users/dhsung/bai-dev/src/ai/backend/agent/docker/kernel.py", line 375, in prepare_krunner_env_impl
    |     .read_text()
    |   File "/Users/dhsung/.pyenv/versions/3.11.2/lib/python3.11/pathlib.py", line 1058, in read_text
    |     with self.open(mode='r', encoding=encoding, errors=errors) as f:
    |     ^^^^^^^^^^^^^^^^^
    |   File "/Users/dhsung/.pyenv/versions/3.11.2/lib/python3.11/pathlib.py", line 1044, in open
    |     return io.open(self, mode, buffering, encoding, errors, newline)
    |     ^^^^^^^^^^^^^^^^^
    | FileNotFoundError: [Errno 2] No such file or directory: '/Users/dhsung/bai-dev/dist/export/python/virtualenvs/python-default/3.11.2/lib/python3.11/site-packages/ai/backend/krunner/alpine/krunner-version.musl_1_2.txt'
    +------------------------------------
2023-02-13 16:41:19.407 INFO ai.backend.agent.server [81252] exit.
```
<img width="1904" alt="image" src="https://user-images.githubusercontent.com/20002/218399465-f1ecc101-b456-44ba-9c04-2eda4b53cf51.png">

After I checked the log, I'm checking the  `/Users/dhsung/bai-dev/dist/export/python/virtualenvs/python-default/3.11.2/lib/python3.11/site-packages/ai/backend/krunner/alpine` directory.

```console
➜  alpine git:(setup/upgrade-to-py3.11) ✗ ls
__init__.py                              krunner-python.musllinux_1_2.txt         ttyd_linux.aarch64.bin
__pycache__                              krunner-version.musllinux_1_2.txt        ttyd_linux.x86_64.bin
krunner-env.musllinux_1_2.aarch64.tar.xz krunner-wheels.musllinux_1_2.dockerfile  versions.txt
krunner-env.musllinux_1_2.dockerfile     plugin.py
krunner-python.musllinux_1_2.dockerfile  requirements.txt
```

Error log shows that  ` FileNotFoundError: ***/krunner-version.musl_1_2.txt`. However, target directory's file name is `krunner-version.musllinux_1_2.txt`. 

So, I think, krunner file pattern is wrong.